docs/index.rst

```diff
@@ -2,74 +2,162 @@
 Dask-Jobqueue
 =============
 
-*Easy deployment of Dask Distributed on job queuing systems like
-PBS, Slurm, and SGE.*
+*Easily deploy Dask on job queuing systems like PBS, Slurm, MOAB, and SGE.*
 
-Motivation
-----------
 
-1. While ``dask.distributed`` offers a flexible library for distributed computing
-   in Python, it is not always easy to deploy on systems that use job queuing
-   systems. Dask-jobqueue provides a Pythonic interface for deploying and
-   managing Dask clusters.
-2. In practice, deploying distributed requires customization, both for the
-   machine(s) that it will deployed on and for the specific application it will
-   be deployed for. Dask-jobqueue provides users with an intuitive interface for
-   customizing dask clusters.
+The Dask-jobqueue project makes it easy to deploy Dask on common job queuing
+systems typically found in high performance supercomputers, academic research
+institutions, and other clusters.  It provides a convenient interface that is
+accessible from interactive systems like Jupyter notebooks, or batch jobs.
 
 Example
 -------
 
 .. code-block:: python
 
    from dask_jobqueue import PBSCluster
+   cluster = PBSCluster()
+   cluster.scale(10)         # Ask for ten worker nodes
 
-   cluster = PBSCluster(processes=6, threads=4, memory="16GB")
-   cluster.start_workers(10)
+   # wait for jobs to arrive, depending on the queue, this may take some time
```

Comment:
Wait can be done after the client creation (as you very well known), I often run a cell with client on Jupyter to know when the worker are connected. But maybe it is better for user understanding to put it here.

Reply:
I've moved this line down below `client = Client(...)`.  I started writing "If you are in a Jupyter notebook then you might want to render `cluster` so that you can see the status of the cluster" but then I realized that this is the header example of the entire project, and so I figured that this was too much detail upfront.